### PR TITLE
Update ANU

### PR DESCRIPTION
**Pull request by hosking**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -13569,6 +13572,7 @@ Neucimar Jerônimo Leite,UNICAMP,http://www.ic.unicamp.br/~neucimar,Qfothk4AAAAJ
 Nevil Brownlee,University of Auckland,https://www.cs.auckland.ac.nz/people/n-brownlee,t7vOIzMAAAAJ
 Neville Churcher,University of Canterbury,http://www.cosc.canterbury.ac.nz/neville.churcher,NOSCHOLARPAGE
 Neville I. Churcher,University of Canterbury,http://www.cosc.canterbury.ac.nz/neville.churcher,NOSCHOLARPAGE
+Neville Grech,Australian National University,https://www.nevillegrech.com,QP3gdYAAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:13575]: Hadn't realized Neville was going to ANU! Might advise him to update his homepage...

File: csrankings.csv [hunk 2]
```diff
@@ -16357,6 +16362,7 @@ Sabine Glesner,TU Berlin,https://www.sese.tu-berlin.de/menue/ueber_uns/team/sabi
 Sabine Schulte im Walde,University of Stuttgart,https://www.ims.uni-stuttgart.de/institut/mitarbeiter/schulte,kObRYIAAAAAJ
 Sabine Süsstrunk,EPFL,http://ivrl.epfl.ch/people/susstrunk,EX3OYP4AAAAJ
 Sabine Timpf,University of Augsburg,https://www.uni-augsburg.de/en/fakultaet/fai/geo/prof/geoinf/geoinf-team/s-timpf,LT_XljUAAAAJ
+Sabrina Caldwell,Australian National University,
```
Comment (emeryberger) [inline csrankings.csv:16365]: Needs a home page and a Google scholar id (or NOSCHOLARPAGE).

File: csrankings.csv [hunk 3]
```diff
@@ -19082,9 +19088,9 @@ Vanderlei Bonato,USP-ICMC,http://lcr.icmc.usp.br/web/members/vanderlei-bonato/in
 Vanessa Braganholo,UFF,http://www2.ic.uff.br/~vanessa,QT9-AZ4AAAAJ
 Vanessa Braganholo Murta,UFF,http://www2.ic.uff.br/~vanessa,QT9-AZ4AAAAJ
 Vanessa Frías-Martínez,University of Maryland - College Park,https://www.cs.umd.edu/people/vfrias,R0L7Y50AAAAJ
-Vanessa J. Teague,University of Melbourne,https://people.eng.unimelb.edu.au/vjteague,3CVS_xYAAAAJ
+Vanessa J. Teague,Australian National University,https://cs.anu.edu.au/people/vanessa-teague,3CVS_xYAAAAJ
 Vanessa P. Braganholo,UFF,http://www2.ic.uff.br/~vanessa,QT9-AZ4AAAAJ
-Vanessa Teague,University of Melbourne,https://people.eng.unimelb.edu.au/vjteague,3CVS_xYAAAAJ
+Vanessa Teague,Australian National University,https://cs.anu.edu.au/people/vanessa-teague,3CVS_xYAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:19093]: Page says "honorary"; is this because this person hasn't joined yet? What's their join date?

File: csrankings.csv [hunk 4]
```diff
@@ -8503,6 +8504,7 @@ Jennifer Waycott,University of Melbourne,http://findanexpert.unimelb.edu.au/disp
 Jennifer Widom,Stanford University,http://infolab.stanford.edu/~widom,zdKmnYwAAAAJ
 Jenny Coady,Heriot-Watt University,https://www.hw.ac.uk/staff/uk/macs/jenny-coady.htm,NOSCHOLARPAGE
 Jenny Waycott,University of Melbourne,http://findanexpert.unimelb.edu.au/display/person52243,kc5Fr-0AAAAJ
+Jennyfer Lawrence Taylor,Australian National University,https://jennyferlawrencetaylor.com,YojqErAAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:8507]: No mention of ANU on home page, not an ANU page.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by emeryberger** [comment]
Bump @hosking

**Comment by emeryberger** [comment]
Closing for lack of response. Please feel free to reopen.

File: csrankings.csv (hunk 1 above)
Comment (hosking) [reply csrankings.csv:13575]: Yep, he has accepted an offer but waiting for borders to open.

File: csrankings.csv (hunk 4 above)
Comment (hosking) [reply csrankings.csv:8507]: Just started at ANU.

File: csrankings.csv (hunk 3 above)
Comment (hosking) [reply csrankings.csv:19093]: She has no other University affiliation, independently wealthy, so eligible for ARC grants and also to supervise PhD students at ANU.

**Pull request closed without merging**